CONTRIBUTING.md

```diff
@@ -23,6 +49,81 @@ For feature requests, please:
 - Explain your motivation for the feature
 - List any alternatives you've considered
 
+## Branch Rules
+
+- `main` - Production-ready code, protected branch
+- `develop` - Integration branch for features
+- `feature/*` - New features
+- `release/*` - Release preparation
+- `bugfix/*` - Bug fixes for development
+- `hotfix/*` - Emergency production fixes
+
+## Version Control
+
+- Semantic versioning: `vMAJOR.MINOR.PATCH`
+- Release tags on `main` branch only
+- Package versions match git tags
+
+## Release Process
+
+week one for `release/v4.1.0`
+
+- Planning meeting for `release/v4.1.0` with release scope and milestone objectives set by the maintainers. Release and maintainer meeting agendas and schedules are posted on the project repository [wiki](https://github.com/AIHawk/AIHawk/wiki) and shared in the `#releases` channel on Discord.
+- `release/v4.0.0` release candidate ready for release
+- `release/v4.0.0` merge into `develop`, `main`
+- tag `main` as `release/v4.0.0`
+- `release/v4.0.0` published to AIHawk/releases and PyPI as a package with release documentation
+- delete `release/v4.0.0` branch
+
+release/v4.1.0 release weeks
+
+- Contributers work on issues and PRs, prioritizing next milestone
+- Maintainers review PRs from `feature/*`, `bugfix/*` branches and issues, merging into `develop`
+- Maintainers review PRs from `hotfix/*` branches and issues, merged into `main` and `develop`, `main` tagged and merged into `4.0.1` package and `release/v4.0.1` and `release/v4.1.0`, documentation is updated
+
+last week, release candidate
+
+- `develop` is frozen, only bug fixes
+- create release branch `release/v4.1.0` from `develop`
+- only bug fixes are merged into `release/v4.1.0`
+- additional testing and release candidate review
+
+week one is repeated for `release/v4.2.0`
+
+## Roles
+
+### Organization Owner
+
+- Has full access to all repositories
+- Controls organization-wide settings and permissions
+- Can set base permissions for all members
+- Manages repository settings and collaborator access
+
+### Release Manager
+
+- Creates and manages release branch from develop
+- Coordinates release cycles and versioning
+- Merges release into main
+
+### Maintainer
```

Comment:
can you add to maintainer
- triage issues, bugs, PRs

Reply:
this is good. will update the section.